blockstack/lib/nameset/virtualchain_hooks.py

```diff
@@ -207,7 +207,7 @@ def db_parse( block_id, txid, vtxindex, op, data, senders, inputs, outputs, fee,
         
    # check if the v2 upgrade threshold block has already been reached.
```

Comment:
Can you update this comment and the log message below as well to avoid confusion? Otherwise LGTM.